frontend/simple/views/components/Voting/Voting.vue

```diff
@@ -169,47 +272,43 @@ export default {
           break
       }
 
-      if (!this.proposal.votes) {
-        return this.proposal.ownProposal
+      if (this.votes.received === 0) {
+        return this.isOwnProposal
```

Comment:
> the proposal gets 1 'for' vote when proposed (the proposer's vote), so 2 here means the proposer + 1 :) maybe I can subtract the proposer's vote from the received votes and go back to 1 here, that would be more self-explanatory? - @hubudibu 

If now it's zero again, shouldn't it be 1 following your explanation? And if it's actually zero, it can be just `if (!this.votes.received)`